modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/PlatformTestNodeRunner.java

```diff
@@ -271,7 +271,7 @@ private static void createTables(Ignite node) {
                 SchemaConfigurationConverter.convert(schTbl, tblCh)
         ));
 
-        int maxTimePrecision = TemporalColumnType.MAX_TIME_PRECISION;
+        int maxTimePrecision = CatalogUtils.MAX_TIME_PRECISION;
```

Comment:
but TemporalColumnType.MAX_TIME_PRECISION; still used, why do we need it ?

Reply:
Reverted.